service/src/main/java/com/codeforcommunity/requester/MapRequester.java

```diff
@@ -51,6 +54,24 @@ public void updateBlocks(List<String> blockIds, BlockStatus updateTo) {
     updateLayers(blockIds, updateTo, this.tokenFuture);
   }
 
+  /**
+   * Creates a JSONObject based off the private streets geoJSON file.
+   *
+   * @return the private street JSONObject
+   */
+  public JsonObject getPrivateStreets() {
+    try {
+      String privateStreetJson =
+          new String(
+              Files.readAllBytes(
+                  Paths.get("service/src/main/resources/private_streets_light.geojson")));
+      return new JsonObject(privateStreetJson);
+    } catch (Exception e) {
+      logger.error("There was an error loading private street GeoJSON data", e);
+      return new JsonObject();
```

Comment:
We might want to throw a custom exception here as opposed to an empty JSON object. Exceptions percolate up to `FailureHandler`, where we can return a custom HTTP error. This would be 500-level, there might already be a custom exception which fits this case nicely.

Reply:
Because of the log we'll know anyways, so not necessary, but it'd be good for whatever route calls this to not silently return a 200.